app/controllers/public/index.js

```diff
@@ -6,10 +6,14 @@ export default Controller.extend({
 
   speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
We'll actually be needing two computed properties. featuredSpeakers and nonFeaturedSpeakers based on one speakers array.